Signin.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Sign in</title>
+</head>
+<style>

```

Comment:
Can we please have css in .css file for signin or login page instead of having it in html file.